ci/Jenkinsfile

```diff
@@ -44,15 +44,14 @@ pipeline {
                         timeout(time: 4, unit: 'HOURS')
                     }
                     environment {
-                        PARALLEL_LEVEL = '10'
+                        PARALLEL_LEVEL = '16'
```

Comment:
Is this always going to be 16? Is there no environment variable set by the Jenkins runtime? How do they do it in RMM?

Reply:
I think that would be fine.